app/gui/src/dashboard/components/AriaComponents/Dialog/Popover.tsx

```diff
@@ -96,6 +99,7 @@ export function Popover(props: PopoverProps) {
   const popoverStyle = { zIndex: '' }
 
   return (
+    // @ts-expect-error placement is optional, but destructure it to make it either value or undefined, and ts complains
```

Comment:
i would prefer `!` with an eslint suppression instead because it's slightly more fine-grained.

but yeah this is a problem with react-aria-components T_T

Reply:
This library became strictly compliant only in the latest release (!). Here’s the link: https://arc.net/l/quote/ouvzotbc

I hope they support `exactOptionalPropertyTypes` someday. LoL